scripts/jenkins/qa_devstack.sh

```diff
@@ -157,4 +157,11 @@ tox -e smoke
 EOF
 fi
 
+if [ -z "${DO_UNSTACK}" ]; then
+    sudo -u stack -i <<EOF
+cd $DEVSTACK_DIR
+./unstack.sh
+EOF
+fi
```

Comment:
Actually the unstack is added to prevent an error :)
If not done and swift runs in foreground (I did not understand yet why this is) then swift grabs the (ssh) terminal and will keep the ssh connection open. Thus the job will never finish.

So from this POV we should care about errors - but in a way that all swift processes are killed, because otherwise the jobs will hang forever again.

So maybe a better solution would have been to run `qa_devstack.sh` in a detached screen session instead of unstacking it. Because then the error could be ignored and the terminal grabbing (in this case screen) of swift could be ignored.

@sjamgade I guess the screen solution is better than this approach.